src/app/components/universal/newSafetyDocumentUpdate/newSafetyDocumentUpdate.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+
+@Component({
+  selector: 'app-newSafetyDocumentUpdate',
+  templateUrl: './newSafetyDocumentUpdate.component.html',
+  styleUrls: ['./newSafetyDocumentUpdate.component.css']
+})
+export class NewSafetyDocumentUpdateComponent implements OnInit{
+
+  safetyDocumentUpdated: boolean;
+
+  @Input() heading: string;
+  @Input() body: string;
+  @Input() alertType: string; // Should be either warning, ??? or ???
```

Comment:
Are the ??? placeholders? And where are these inputs used? I see them assigned (in a method that also doesn't seem to be used) but I don't see where they're accessed.